socialdistribution/urls.py

```diff
@@ -12,7 +12,8 @@
     # url(r'^blog/', include('blog.urls')),
 
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^friends/', include('author.urls')),
+    #Gives an error when uncommented likely because there is no urls.py in author app 
+    #url(r'^friends/', include('author.urls')),
```

Comment:
No, don't thijnk /login has to be login exempt. Was trying it out yesterday and didn't encounter a problem with it